### DISCUSSION TITLE
Bell TV

### DISCUSSION THREAD

**Discussion by Qlogic711** [fabianishere/udm-iptv#345] [General]
I have set this up again, and everything looks good but the after a new seconds I lose picture again.

here is my udm-iptv diag

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth9
WAN VLAN: 36 (dev iptv)
WAN DHCP: true (options "-n -o")
WAN Ranges: 10.2.0.0/16 10.238.0.0/16
LAN Interfaces: br40
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
71: iptv@eth9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.238.130.204/22 scope global iptv
       valid_lft forever preferred_lft forever
10.2.0.0/16 via 10.238.128.1
10.238.0.0/16 via 10.238.128.1
10.238.128.0/22 proto kernel scope link src 10.238.130.204
=== Service Logs ===
Aug 09 13:49:02 Hollingsworth udm-iptvd[3509851]: Obtaining IP address for VLAN interface
Aug 09 13:49:02 Hollingsworth udm-iptvd[3509885]: udhcpc: started, v1.30.1
Aug 09 13:49:02 Hollingsworth udm-iptvd[3509885]: udhcpc: sending discover
Aug 09 13:49:02 Hollingsworth udm-iptvd[3509885]: udhcpc: sending select for 10.238.130.204
Aug 09 13:49:02 Hollingsworth udm-iptvd[3509885]: udhcpc: lease of 10.238.130.204 obtained, lease time 603523
Aug 09 13:49:03 Hollingsworth udm-iptvd[3509851]: NATing IPTV network ranges (if necessary)
Aug 09 13:49:03 Hollingsworth udm-iptvd[3509851]: Creating static routes (if necessary)
Aug 09 13:49:03 Hollingsworth udm-iptvd[3509851]: Setting up IGMP Proxy
Aug 09 13:49:03 Hollingsworth udm-iptvd[3509851]: Using improxy...
Aug 09 13:49:03 Hollingsworth udm-iptvd[3509851]: Starting IGMP Proxy

only thing i see that is i weird is this 

10.238.128.0/22 proto kernel scope link src 10.238.130.204

everyone elses diag doesn't have this.

any thoughts?

**Comment by dP210** [comment]
I'm actually trying to do something similar this weekend. 

I can't seem to get an IP link for longer than a few seconds.  I get the error 619 consistently. Any ideas what that may be pointing to? I'm in the GTA.



Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth9
WAN VLAN: 36 (dev iptv)
WAN DHCP: true (options "-o")
WAN Ranges: 10.2.0.0/16 10.125.0.0/16 10.117.0.0/16
LAN Interfaces: br100
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
19298: iptv@eth9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN group default qlen 1000
    inet 10.117.166.X/20 scope global iptv
       valid_lft forever preferred_lft forever
10.117.160.0/20 proto kernel scope link src 10.117.166.X
=== Service Logs ===
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: DEBUG[init_interface@195]: if_index = 18007
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: DEBUG[init_interface@208]: if_mtu = 1500
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: DEBUG[init_interface@214]: if_falgs = 4163
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: DEBUG[init_interface@178]: iptv's ipv4 address is 10.117.166.X
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: DEBUG[init_interface@195]: if_index = 19298
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: DEBUG[init_interface@208]: if_mtu = 1500
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: DEBUG[init_interface@214]: if_falgs = 4163
Aug 09 21:51:20 Dream-Machine-Pro udm-iptvd[3991790]: ERROR[main@619]: exiting.........

**Comment by Qlogic711** [comment]
I am not sure what is going on. I had it working on a edge router but decided to try this. But would lose picture after a couple minutes. I put back the Edge router with no changes and getting the same thing. Something changed but not sure what

i do notice you are getting the same thing like me

10.117.160.0/20 proto kernel scope link src 10.117.166.37

the ip subnets don't match. they used to but not anymore
________________________________

**Comment by dP210** [comment]
I too had an Edgerouter set up for the longest time but got myself a UDM Pro a few months back and only recently stumbled upon Fabian's solution. 

So this morning I played with it a bit more and was able to get it up and running.

The one thing I think may have helped was after I read this [thread](https://www.reddit.com/r/bell/comments/l7usok/replacing_home_hub_3000_with_ubiquiti_unifi/?utm_source=share&utm_medium=ios_app&utm_name=iossmf) from some time ago and the VLAN's needing to be applied to a switch port profile.

So I modified my local VLAN that connected to my STB and applied the Tagged Networks with both VLAN 35 and 36. The way to do that with current UDM OS is switching to the 'old' interface AFAIK.

It's been running for about 2 hours and recorded to PVR. Fingers crossed it stays up!

**Reply by Qlogic711** [reply]
can you post your diag and what you did for the static route

**Reply by Qlogic711** [reply]
I added your tagging, but still losing signal after a couple minute.

**Reply by Qlogic711** [reply]
if you can also post your UDM settings, I am missing something but can't think of it

**Comment by Qlogic711** [comment]
I am getting this

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth9
WAN VLAN: 36 (dev iptv)
WAN DHCP: true (options "-o")
WAN Ranges: 10.2.0.0/16 10.238.0.0/16 10.125.0.0/16
LAN Interfaces: br40
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
82: iptv@eth9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.238.130.204/22 scope global iptv
       valid_lft forever preferred_lft forever
10.2.0.0/16 via 10.238.128.1
10.125.0.0/16 via 10.238.128.1
10.238.0.0/16 via 10.238.128.1
10.238.128.0/22 proto kernel scope link src 10.238.130.204
=== Service Logs ===
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout
Aug 11 13:45:01 Hollingsworth udm-iptvd[1485496]: DEBUG[main@655]: timeout

**Comment by dP210** [comment]
Here's my Config:

For UDM, I have 2 VLAN only networks, 35 and 36 which are tagged on the VLAN for my LAN IPTV connection. In my case VLAN 100.

For the VLAN 100 network, I have also put in the DNS servers I've seen posted [online](https://www.dslreports.com/forum/r33687505-). I probably have extra DNS
<img width="698" alt="Screenshot 2024-08-11 at 3 52 10 PM" src="https://github.com/user-attachments/assets/90d2ac7e-278d-4504-b36a-455a5980498a">
 servers in there, and could clean up.

Of course IGMP snooping enabled on the VLAN 100 network.

It is weird that there is the mismatch of the subnet for the static routes. I'm hoping it stays up with a reboot which I've yet to try.

Also I did connect the HH3000 and hardwired the 4K receiver last night while I was fiddling with the settings to get the receiver back on LAN connectivity since it was trying to establish via WiFi (WPS setting screen was showing).

=== Configuration ===
WAN Interface: eth9
WAN VLAN: 36 (dev iptv)
WAN DHCP: true (options "-o")
WAN Ranges: 10.2.0.0/16 10.125.0.0/16 10.117.0.0/16
LAN Interfaces: br100
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
30178: iptv@eth9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.117.166.X/20 scope global iptv
       valid_lft forever preferred_lft forever
10.2.0.0/16 via 10.117.160.1 
10.117.160.0/20 proto kernel scope link src 10.117.166.X
10.125.0.0/16 via 10.117.160.1

**Reply by Qlogic711** [reply]
weird this is my UDM

![image](https://github.com/user-attachments/assets/54e0d259-6684-4291-9ca8-425a8ea6ddd1)

![image](https://github.com/user-attachments/assets/cb027711-2c65-4f1a-a4b4-b68098e7f516)

![image](https://github.com/user-attachments/assets/97640cf5-7b66-4ffc-bb39-ae6c1a28e21e)

![image](https://github.com/user-attachments/assets/430eb9a0-71e7-4e84-9876-c78ea74a8ec1)

![image](https://github.com/user-attachments/assets/28df4cd0-197a-4ed7-939c-fc3c08b8b0b8)

**Reply by Qlogic711** [reply]
can you show me what you put for this

For the VLAN 100 network, I have also put in the DHCP servers I've seen posted [online](https://www.dslreports.com/forum/r33687505-). I probably have extra DHCP servers in there, and could clean up.

**Comment by dP210** [comment]
I've updated my post with the DNS servers.

Also you need to put the VLAN 35/36 on the **Switch Port** profile in the legacy UI. 
<img width="953" alt="Screenshot 2024-08-11 at 3 56 45 PM" src="https://github.com/user-attachments/assets/a33118d9-9fce-435c-a0bd-2f3e1a58a3e5">

**Reply by Qlogic711** [reply]
i created a profile 

![image](https://github.com/user-attachments/assets/65854f41-49e3-45d6-8b36-ac7931433602)

still only works for a couple seconds

**Reply by Qlogic711** [reply]
got it working

needed to turn on IGMP snooping on the vlan 35 and 36 

![image](https://github.com/user-attachments/assets/a563314c-1714-4f47-9d28-e42463c3e65b)

![image](https://github.com/user-attachments/assets/29c7ef85-1fb4-4536-9b7a-e8b31a051b71)

I knew is was something stupid i was missing

**Reply by dP210** [reply]
> i created a profile
> 
> ![image](https://private-user-images.githubusercontent.com/99082953/356903213-65854f41-49e3-45d6-8b36-ac7931433602.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM0MDkwNjAsIm5iZiI6MTcyMzQwODc2MCwicGF0aCI6Ii85OTA4Mjk1My8zNTY5MDMyMTMtNjU4NTRmNDEtNDllMy00NWQ2LThiMzYtYWM3OTMxNDMzNjAyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDIwMzkyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkZDA1MmZlZTVjYTFjMmJkZTU2ZDU5ODVlZTVlNTgzMWRhZmI1ZmQ1MjZjY2JlY2Q3NmZkM2YyODE4YWUzMGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.llMKVQcOKaG_l4WGbZ2ieKXe6a2adsI9Z_Q4u0MK8lQ)
> 
> still only works for a couple seconds

Awesome! Hoping this stays working for an extended period of time and works through reboots. 

Fingers crossed.

**Comment by dP210** [comment]
@Qlogic711 Are you finding that over time the device will freeze for about 1.5 mins every hour or so? I find that to happen on my live TV as well as PVR'd content.

**Comment by Qlogic711** [comment]
No issues I have seen. Have you updated your udm?
